azure/configuration/az-naming-standards.ps1

```diff
@@ -0,0 +1,13 @@
+#Todo: How do we allow overrides of configuration from deployment?
+Class AzNamingStandards{
```

Comment:
Added this is the Azure configuration.  I did add a todo at some point it woudl be nice to figure out how we can override theses naming standards or resource type abbreviation in our own configuration file within a deployment if we wanted to after importing the modules.